This patch comes from in-house, to deal with SSLv2 no longer being supported.
As it is Solaris-specific, it will not be passed upstream.

--- Python-2.7.10/Lib/test/test_ssl.py.orig	2015-07-24 14:34:42.067600359 -0700
+++ Python-2.7.10/Lib/test/test_ssl.py	2015-08-03 11:48:02.934532909 -0700
@@ -25,6 +25,9 @@
 ssl = support.import_module("ssl")
 
 PROTOCOLS = sorted(ssl._PROTOCOL_NAMES)
+if hasattr(ssl, 'PROTOCOL_SSLv2'):
+    delattr(ssl, 'PROTOCOL_SSLv2')
+    PROTOCOLS.remove(0)
 HOST = support.HOST
 
 def data_file(*name):
